lib/galaxy/managers/workflows.py

```diff
@@ -268,9 +268,11 @@ def get_stored_workflow(self, trans, workflow_id, by_stored_id=True) -> StoredWo
                     )
                 )
         stored_workflow = workflow_query.options(
-            joinedload("annotations"),
-            joinedload("tags"),
-            subqueryload("latest_workflow").joinedload("steps").joinedload("*"),
+            joinedload(trans.app.model.StoredWorkflow.annotations),
+            joinedload(trans.app.model.StoredWorkflow.tags),
+            subqueryload(trans.app.model.StoredWorkflow.latest_workflow)
+            .joinedload(trans.app.model.Workflow.steps)
```

Comment:
You can use model.* instead of trans.app, right ?

Reply:
Oh, absolutely! There were a few other similar cases - I thought it might be better to group all those changes in a "cleanup" PR *after* I'm done with all the relevant SA 2.0 updates to Query/select() not to mix them with strictly functional stuff in case something breaks and we need to bisect lots of commits, etc.